server/server.go

```diff
@@ -239,12 +242,14 @@ func NewServer(cfg *config.Config, driver IDriver) (*Server, error) {
 					go s.forwardUnixSocketToTCP()
 				}
 			}
+		} else if runInGoTest && s.cfg.Port == 0 {
```

Comment:
Why this logic is in `else` which means `net.Listen` gets an error?

Reply:
fixed